frontend/src/slices/articles/articles.slice.ts

```diff
@@ -4,15 +4,22 @@ import { DataStatus } from '~/libs/enums/enums.js';
 import { type ValueOf } from '~/libs/types/types.js';
 import { type ArticleWithAuthorType } from '~/packages/articles/articles.js';
 
-import { createArticle, fetchAll, fetchOwn } from './actions.js';
+import {
+  createArticle,
+  fetchAll,
+  fetchOwn,
+  fetchSharedArticle,
+} from './actions.js';
 
 type State = {
   articles: ArticleWithAuthorType[];
+  sharedArticle: ArticleWithAuthorType[] | null;
```

Comment:
Don't now why it could be an array if only single article may be shared at once
```suggestion
  sharedArticle: ArticleWithAuthorType | null;
```